CoreHtmlToImage/HtmlConverter.cs

```diff
@@ -10,28 +11,44 @@ namespace CoreHtmlToImage
     /// </summary>
     public class HtmlConverter
     {
-        private const string toolFilename = "wkhtmltoimage.exe";
-        private static readonly string directory;
+        private const string winToolFileName = "wkhtmltoimage.exe";
+        private const string linuxToolFileName = "/usr/local/bin/wkhtmltoimage";
+        private static  readonly string directory;
         private static readonly string toolFilepath;
 
         static HtmlConverter()
         {
-            directory = AppContext.BaseDirectory;
-            toolFilepath = Path.Combine(directory, toolFilename);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                directory = AppContext.BaseDirectory;
+                toolFilepath = Path.Combine(directory, winToolFileName);
+                CopyFile(winToolFileName);
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                directory = AppContext.BaseDirectory;
+                toolFilepath = linuxToolFileName;
```

Comment:
So this means, for linux users we assume the tool will exist in that specific place? Can't we copy it from embedded resources just like we do for windows?